packages/search-in-workspace/src/browser/search-in-workspace-preferences.ts

```diff
@@ -53,6 +53,11 @@ export const searchInWorkspacePreferencesSchema: PreferenceSchema = {
             description: nls.localizeByDefault('Search case-insensitively if the pattern is all lowercase, otherwise, search case-sensitively.'),
             default: false,
             type: 'boolean',
+        },
+        'search.followSymlinks': {
+            description: nls.localizeByDefault('Controls whether to follow symlinks while searching.'),
+            default: false,
```

Comment:
I believe it should be `true` by default in in vscode:
- https://github.com/microsoft/vscode/blob/596c835be231972bd1d86d243cdb4e7f78cac807/src/vs/workbench/contrib/search/browser/search.contribution.ts#L905-L908